unity_parent_app/api/student_profile.py

```
@@ -20,6 +20,8 @@ def create_otp(email):
     cache = frappe.cache()
     key = "otp_" + email
     otp = generate_otp()
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)